src/containers/FilmFrontpage/AllMoviesAlphabetically.tsx

```diff
@@ -125,12 +77,51 @@ const groupMovies = (movies: MovieType[]) => {
     }
     return acc;
   }, {});
+
   return Object.entries(grouped).map(([letter, movies]) => ({
     letter,
     movies,
   }));
 };
 
+const LoadingShimmer = () => {
+  return (
+    <HStack justify="center">
+      <OneColumn wide>
+        {["#", "Å", "Ø", "Æ"].map((letter, idx) => {
+          return (
+            <MovieGroup key={`Loading-${idx}`}>
+              <LetterHeading textStyle="title.medium" fontWeight="bold" asChild consumeCss>
+                <h2>{letter}</h2>
+              </LetterHeading>
+              {new Array(4).fill(0).map((_, idx2) => {
+                return (
+                  <StyledSafeLink to="" disabled={true} key={idx2}>
+                    <Skeleton css={{ width: "surface.3xsmall", minWidth: "surface.3xsmall", height: "75px" }} />
+                    <MovieTextWrapper>
+                      <Skeleton css={{ marginBottom: "xxsmall", width: "surface.xsmall" }}>
+                        <Heading textStyle="title.small" asChild consumeCss data-title="">
+                          <h3>Example title</h3>
+                        </Heading>
+                      </Skeleton>
+                      <Skeleton css={{ width: "surface.medium" }}>
+                        <Text textStyle="body.small">
+                          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut
+                          labore et dolore magna aliqua.
```

Comment:
Kanskje bare kjøre &nbsp; som alle andre steder? Hvis du skal treffe to linjer kan du like så godt bruke to text-elementer.